python/hail/dataset.py

```diff
@@ -2499,6 +2500,30 @@ def ld_prune(self, r2=0.2, window=1000000, memory_per_core=256, num_cores=1):
 
     @handle_py4j
     @requireTGenotype
+    def ld_matrix(self):
+        """Creates the LD Matrix for the variants in this VDS.
+        
+        **Examples**
+        
+        >>> ld_mat = vds.ld_matrix()
+        
+        Each entry (i, j) in the LD Matrix gives the :math:`R` value between variants i and j. :math:`R` is defined as 
+        `Pearson's correlation coefficient <https://en.wikipedia.org/wiki/Pearson_correlation_coefficient>`__ 
+        :math:`{\\rho}_{x_i,x_j}` between the two genotype vectors :math:`{\\mathbf{x_i}}` and :math:`{\\mathbf{x_j}}`.
+
+        .. math::
+
+            {\\rho}_{x_i,x_j} = \\frac{\\mathrm{Cov}(X_i,X_j)}{\\sigma_X_i \\sigma_Y_j}
+
+        
```

Comment:
Add some ..caution:: about filtering the vds to a reasonable number of variants first.